docs/blog/newsletter-september-2023.rst

```diff
@@ -0,0 +1,111 @@
+- `Why it’s worth it to invest in internal docs <https://increment.com/documentation/why-investing-in-internal-docs-is-worth-it/>`__
+
+--------------------------
+Numbering Section Headings
+--------------------------
+
+To number or not was a question posed in recent months. In general, these queries related to printed (or PDF) manuals.
+
+Often reasons for numbering sections relate to using the number instead of the section title as a reference or to clarify about sequencing (or relationships) between sections. As one documentarian noted, "Titles alone do not tell users ... where they will find the information unless they are already ... familiar with the content structure of the document. When you have to check section 3, you know that it is before section 4 and ... after section 2."
+
+Certain domains may also prefer or require section numbers. For example, if your readers are in a scientific community, a legal or academic environment, or a strictly regulated domain, they may expect to see section numbers.
+
+One caution: if your documentation tool doesn't support automatic numbering, then including numbers can be a maintenance nightmare. If your tool doesn't support automatic numbering, perhaps a plug-in may be needed. 
+
+Several people mentioned issues about updating the content and adding in new sections. Even with automatic numbering, readers of previous versions may suggest looking at a specific section by its number – only to find that the number has changed in a recent update.
+
+When numbering is a content convention, it may not represent a sequence to follow. In this case, numbers can interrupt the reader's focus by forcing them determine whether they really need to go through content sequentially. One person wrote, "Numbering is often one of those unnecessary things that people do without really thinking why." Documentarians may have section 3 before section 4 just because that's the sequence that they thought of – not because it needs to come first.
+
+For online documentation, people were less in favor of using numbers. The discussion emphasized that it could depend on the structure of the online content and navigational features.
+
+------------------------------------------------------
+Dealing with Unpleasant People in Open-Source Projects
+------------------------------------------------------
+
+Although open-source groups are famed for their collaborative attitude, dealing with problematic personalities can sometimes be challenging. A recent WtD discussion provided useful ideas for efficiently dealing with such circumstances.
+
+In this conversation, advice on how to deal with an individual whose behavior in an open-source project was troublesome. This person acted in ways that on the verge of being unacceptable without being overtly prohibited. Their perceived importance in the community added to the complication.
```

Comment:
I think the first sentence is incomplete - there seems to be something missing about the advice in the conversation. Or does it mean that the advice was troublesome?

Reply:
Good point. 🤦 
```suggestion
In this conversation, advice was needed on how to deal with an individual whose behavior in an open-source project was troublesome. This person acted in ways that on the verge of being unacceptable without being overtly prohibited. Their perceived importance in the community added to the complication.
```